app/views/fae/application/_header.slim

```diff
@@ -3,10 +3,10 @@ header.main-header#js-main-header
   h1: a href=root_path = @option.title
 
   - if @fae_topnav_items.present?
-    nav.js-nav
-      ul.main-header-nav
+    nav
+      ul.main-header-nav.js-nav data-coordinates=@fae_navigation.coordinates[0..1].join(',')
```

Comment:
Hmm, you weren't able to package it into a helper? For me, the logic is harder to follow when it's split between Ruby and JS and it should be slightly more performant if it's Ruby-only. I would prefer that route.